### PR TITLE
Port ObsTabContent

### PR DESCRIPTION
**Pull request by cquiroz**
Port the class to a functional component and to use the `useResizeDetector` hook

### CODE REVIEW

File: explore/src/main/scala/explore/tabs/ObsTabContents.scala [hunk 1]
```diff
@@ -192,315 +190,293 @@ object ObsTabContents {
...
-            .flatMap(v => $.modStateIn[IO](State.fovAngle.replace(v)))
-            .runAsyncAndForget
-        }.getOrEmpty
+            )
+            .toList
+        )
       }
+    }(obsView)
+
+  // TODO Use this method
```
Comment (toddburnside) [inline explore/src/main/scala/explore/tabs/ObsTabContents.scala:230]: Why aren't we using this now?

**Review by toddburnside — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/Tile.scala [hunk 2]
```diff
@@ -77,7 +77,7 @@ object Tile {
       .withHooks[Props]
       // info ref
       .useRefToVdom[html.Element]
-      .renderWithReuse { (p, infoRef) =>
```
Comment (rpiaggio) [inline common/src/main/scala/explore/components/Tile.scala:80]: Well, the ref to the title area was called `info` to differentiate from the actual tile contents. Maybe `titleRef`?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/Tile.scala (hunk 2 above)
Comment (cquiroz) [reply common/src/main/scala/explore/components/Tile.scala:80]: Ah ok. I thought it was the actual content. I'll revert

File: explore/src/main/scala/explore/tabs/ObsTabContents.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/tabs/ObsTabContents.scala:230]: Because aladin has problems setting the fov on startup

**Pull request merged**